libraries/AP_Math/AP_Math.cpp

```diff
@@ -266,6 +266,11 @@ ftype wrap_2PI(const ftype radian)
     return res;
 }
 
+ftype wrap_neg2PI_2PI(const ftype radian)
```

Comment:
We like to have tests for all of our functions in AP_Math - it's the gtest stuff in libraries/AP_Math/tests and isn't hard to work with

This implementation should also go in the header to save some bytes